quantum/led_matrix/led_matrix.c

```diff
@@ -105,7 +105,11 @@ static effect_params_t led_effect_params = {0, LED_FLAG_ALL, false};
 static led_task_states led_task_state    = SYNCING;
 #if LED_DISABLE_TIMEOUT > 0
 static uint32_t led_anykey_timer;
+static uint32_t led_disable_timeout = LED_DISABLE_TIMEOUT;
```

Comment:
`LED_DISABLE_TIMEOUT` has been renamed  to `LED_MATRIX_TIMEOUT`.  All instances of this needs to be updated.